README.md

```diff
@@ -573,3 +574,175 @@ export class MyAwesomeComponent extends React.PureComponent {
     }
 }
 ```
+## External libraries
+
+### Prefer native js vs lodash
+
+> Reason: lodash is a great library which contains a lot of useful functions but the modern js has the short identical part of this functionality so some lodash functions are excess alternative.
+
+> Don't forget to check necessary browsers compatibility and turn on polyfills if it's needed.
+```javascript
+// BAD
+_.each([1, 2, 3], callback); 
+// GOOD
+[1, 2, 3].forEach(callback); 
+```
+```javascript
+// BAD
+_.filter([1, 2, 3], callback);
+// GOOD
+[1, 2, 3].filter(callback);
+```
+```javascript
+// BAD
+_.compact([1, 2, 3, 0, false, '']);
+// GOOD
+[1, 2, 3, 0, false, ''].filter(Boolean);
+```
+```javascript
+// BAD
+_.map([1, 2, 3], callback);
+// GOOD
+[1, 2, 3].map(callback);
+```
+```javascript
+// BAD
+_.every([1, 2, 3], callback);
+// GOOD
+[1, 2, 3].every(callback);
+```
+```javascript
+// BAD
+_.some([1, 2, 3], callback);
+// GOOD
+[1, 2, 3].some(callback);
+```
+```javascript
+// BAD
+_.reduce([1, 2, 3], callback, 0);
+// GOOD
+[1, 2, 3].reduce(callback, 0);
+```
+```javascript
+// BAD
+_.concat([1, 2, 3], [4, 5, 6]);
+// GOOD
+[1, 2, 3].concat([4, 5, 6]);
+```
+```javascript
+// BAD
+_.indexOf([1, 2, 3], 1);
+// GOOD
+[1, 2, 3].indexOf(1);
+```
+```javascript
+// BAD
+_.includes([1, 2, 3], 1);
+// GOOD
+[1, 2, 3].includes(1);
```

Comment:
`Array.prototype.includes` is not a cross browser and still not supported in IE11. Native variant is only preferred when polyfill is included. Otherwise it dangerous and we should not recommend it.

Reply:
I will add a attention about polyfills